examples/hello_pytorch/htcondor/hello_pytorch_gpu.sub

```diff
@@ -1,11 +1,11 @@
 # hello_pytorch_gpu.sub
 # Submit file to access the GPU via docker
 
-# Must set the universe to Docker
-universe = docker
+# Must set the universe to 'container' to use Docker
+universe = container
 # To avoid excessive pulls, and potential rate limits, set as "missing" (default value) unless "always" needed for updated tag
 docker_pull_policy = missing
```

Comment:
This was informed by the email discussion that was captured on https://github.com/UW-Madison-DSI/pixi-docker-chtc/pull/17#discussion_r2150881454 before it got factored out to this PR which is focused on Docker.